v.ts

```diff
@@ -0,0 +1,11 @@
+import axios from "axios";
+import { NowRequest, NowResponse } from "@now/node";
+
+export default async (req: NowRequest, res: NowResponse) => {
+  const link = req.query.link;
+  const a = axios.create();
+  const r = await a.get(
```

Comment:
You don't need to create axios. `axios.get` on his own is fine.

eg. `const { data } = await axios.get("https//face....")`